### DISCUSSION TITLE
Reset Command - question

### DISCUSSION THREAD

**Discussion by proddy** [emsesp/EMS-ESP32#1308] [General]
Hi @MichaelDvP, was there a reason to include a dash as an option in the Reset command? 

in `locale_common.h` `MAKE_ENUM(enum_reset, FL_(dash), FL_(maintenance), FL_(error))`

as `Boiler::set_reset()` doesn't use it and it will throw a command error.

**Comment by MichaelDvP** [comment]
Hmm, afair there are two reasons:
- not showing a value in dashboard, obsolete since we have the CMD type now.
- safety for not accidential send a reset: users have to activ select an option. Sending a reset to often will block the boiler.

Maybe add in Boiler::set_reset a `if (num == 0) return true;` 
But using the reset command and click send without selecting the right option is a user error.

**Reply by proddy** [reply]
ok, makes sense. I'll add the num == 0 check
